include/mata/nfa.hh

```diff
@@ -214,6 +214,9 @@ struct Post : private Util::OrdVector<Move> {
 	}
 };
 
+
+    static Post empty_post;
```

Comment:
I suggest making this `const` (if syntactically/semantically possible), since someone might add something (accidentaly) to this `empty_post` and then we will return non-empty post everytime.

Reply:
ok, now it is there